website/docs/reference/resource-configs/materialize-configs.md

```diff
@@ -5,47 +5,79 @@ id: "materialize-configs"
 
 ## Performance optimizations
 
-### Incremental models
-Materialize, at its core, is a real-time database that delivers incremental view updates without ever compromising on latency or correctness.
-Materialized views are incremental models, defined once. 
+### Clusters
+
+<Changelog>
+
+- **v1.2.0:** Enable the configuration of [clusters](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/CHANGELOG.md#120---2022-08-31).
+
+</Changelog>
+
+The default [cluster](https://materialize.com/docs/overview/key-concepts/#clusters) that is used to maintain materialized views or indexes can be configured in your [profile](/reference/profiles.yml) using the `cluster` connection parameter. To override the cluster that is used for specific models (or groups of models), use the `cluster` configuration parameter.
+
+<File name='my_view_cluster.sql'>
+
+```sql
+{{ config(materialized='materializedview', cluster='not_default') }}
+
+select ...
+```
+
+</File>
+
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  project_name:
+    +materialized: materializedview
+    +cluster: not_default
+```
+
+</File>
+
+
```

Comment:
we have a nifty module specifically for this use case! if you're interested feel free to use it, otherwise we can keep it as it. See [versioning blocks of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) i think it'd look something like this (but GitHub sucks at rendering suggestions that themselves have triple backticks in them. but docusaurus will render it ok
```
<VersionBlock firstVersion="0.21" lastVersion="1.0">

	your stuff

</VersionBlock>
```

```suggestion
<VersionBlock firstVersion="1.2">
### Clusters

<Changelog>

- **v1.2.0:** Enable the configuration of [clusters](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/CHANGELOG.md#120---2022-08-31).

</Changelog>

The default [cluster](https://materialize.com/docs/overview/key-concepts/#clusters) that is used to maintain materialized views or indexes can be configured in your [profile](/reference/profiles.yml) using the `cluster` connection parameter. To override the cluster that is used for specific models (or groups of models), use the `cluster` configuration parameter.

<File name='my_view_cluster.sql'>

```sql
{{ config(materialized='materializedview', cluster='not_default') }}

select ...
```

</File>

<File name='dbt_project.yml'>

```yaml
models:
  project_name:
    +materialized: materializedview
    +cluster: not_default
```

</File>

</VersionBlock>

```